src/Dropdown.js

```diff
@@ -51,10 +52,12 @@ const propTypes = {
 }
 
 const defaultProps = {
-  as: 'div',
   itemSelector: '* > *',
 }
 
+/**
```

Comment:
?

Reply:
Didn't compile otherwise, I need to add docs tho